tomcat/src/main/docker/50-tomcat.bash

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+# Unpack a WAR app (if present) beforehand so that Stackdriver Debugger
+# can load it. This should be done before the JVM for Tomcat starts up.
+export ROOT_WAR=$CATALINA_BASE/webapps/ROOT.war
+export ROOT_DIR=$CATALINA_BASE/webapps/root
```

Comment:
`root` should be all cap like `ROOT` right? otherwise we may have multiple extracts